contrib/test.sh

```diff
@@ -94,3 +94,4 @@ if [ "$DO_BENCH" = true ]; then
     cargo bench --all --features="unstable"
 fi
 
+exit 0
```

Comment:
What is the benefit of explicitly saying exit 0? Isn't that the default?

Reply:
The default exit status is the exit status of the last command, because we use `set -e` to exit the script early if any command fails then it is going to be 0. I like to do it explicitly though.